perf/perf_cpu_hotplug.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+#
+# Copyright: 2021 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.ibm.com>
+
+import os
+import platform
+from avocado import Test
+from avocado.utils import cpu, distro, genio, process
+from avocado.utils.software_manager import SoftwareManager
+from avocado import skipIf
+
+IS_POWER_NV = 'PowerNV' in genio.read_file('/proc/cpuinfo').rstrip('\t\r\n\0')
+
+
+class perf_cpu_hotplug(Test):
+
+    """
+    This tests hv_24x7 and hv_gpci cpu_hotplug
+    :avocado: tags=perf,cpu_hotplug,events
+    """
+    # Initializing fail command list
+    fail_cmd = list()
+
+    def _check_file(self, event_type):
+        self.event_sysfs = "/sys/bus/event_source/devices/%s" % event_type
```

Comment:
any specific reason  variable scope is class level , i feel we should make scope function level only as did not see it use any where else as it increase garbage collector job  what you say ?